### ISSUE TITLE
Differentiate same heroes

### ISSUE THREAD

**Issue by darklordabc** [LegendsOfDota/LegendsOfDota#330]
Not so much of a problem, more like a potential room for improvement. It would be cool if there was a way to visually represent different heroes, in the instances where there is multiples of them (nearly every game). For example, if youre playing a game with 5 miranas, it can be difficult to tell which one is which, and combine that with the fact that heroes are already unpedictable in LoD, it can be inconvenient. 

My only two ideas so far, is giving additional heros a color tint, or having the game give them cosmetics (if this is possible).

**Comment by ash47** [comment]
Any good suggestions on how we can actually differentiate them, in terms of code?

**Comment by iprobie** [comment]
how about a small square or circle next to the player's name with a colour that matches the player's colour slot?

**Comment by Njol** [comment]
How abut the normal colors that dota uses, but LoD hides? (at least in the top bar)

**Comment by Jexah** [comment]
Yeah, that sounds like a good idea. I wish LoD showed those colours up the
top, as well as being able do double click on those icons to select allied
heroes.
